src/main/resources/default/templates/system/info.html.pasta

```diff
@@ -55,7 +55,7 @@
                         <tr>
                             <td>@cookie.name()</td>
                             <td class="text-right">
-                                @cookie.value()
+                                <div style="word-wrap: break-word; word-break: break-all">@cookie.value()</div>
```

Comment:
aren't there bs classes for this? text-break(?)

Reply:
yes, works fine, thx.